### DISCUSSION TITLE
Spring Batch like functionality in micronaut

### DISCUSSION THREAD

**Discussion by raghu298** [micronaut-projects/micronaut-data#2357] [General]
Hi all, is there anything in micronaut to have spring batch like functionality where you have multiple jobs you need to write.
I did not find anything in micronaut.

**Comment by raghu298** [comment]
@dstepanov could you please guide?

**Reply by dstepanov** [reply]
We don't have anything like that. I don't think it's complicated to write, you might want to use Rector for some complicated pipelines.

**Reply by raghu298** [reply]
@dstepanov using reactor can we define the pipeline like spring-batch?
think like suppose you have reader and writer are databases where you want to achieve chunking, retry , job monitoring and all.

**Reply by raghu298** [reply]
@dstepanov newbie to micronaut and reactor too can you give samples to check jobs using reactor?

**Reply by dstepanov** [reply]
You would need to implement it yourself
